tests/v2/validation/snapshot/snapshot_restore_upgrade_strategy_test.go

```diff
@@ -41,23 +41,25 @@ func (s *SnapshotRestoreUpgradeStrategyTestSuite) SetupSuite() {
 
 func (s *SnapshotRestoreUpgradeStrategyTestSuite) TestSnapshotRestoreUpgradeStrategy() {
 	snapshotRestoreK8sVersion := &etcdsnapshot.Config{
-		UpgradeKubernetesVersion:     s.clustersConfig.UpgradeKubernetesVersion,
+		UpgradeKubernetesVersion:     "",
```

Comment:
shouldn't upgrade strategy have a k8s version to test the upgrade? or does this static test depend on the user having upgraded the cluster already?